client/src/App.js

```diff
@@ -56,6 +57,7 @@ export default class App extends Component {
         this.controller.getMessagesForThread(threads[0].id);
       }
     });
+    this.controller.getFromNumbers();
```

Comment:
I'm not familiar with the `grommet` library but I think you might have access to the `componentDidMount` lifecycle hook here. I'd recommend calling this function (fetching the data) inside of that instead of here in the constructor. The react docs have a good example: https://reactjs.org/docs/faq-ajax.html.